bin/FvwmPrompt/meson.build

```diff
@@ -0,0 +1,33 @@
+# Fvwmprompt is written in golang. Unfortunately there is still
+# no automagic in meson to handle golang projects. So we have to
+# do it manually. It'll be fine™
+
+# Go Go Gadget Golang hacks
+go_env = environment()
+# TODO: These LDFLAGS were pulled straight from the legacy Makefile
+#       I'm not sure if they're necessary or not.
+go_env.set('CGO_LDFLAGS', '-s -w')
```

Comment:
Yes, they’re still needed.

Reply:
If you absolutely must handroll this terrible breakage of user environments, meson provides an official `-Dstrip=false` builtin core option. It will handle stripping binaries at install time for you.

The meson option exists specifically so that people who are insistent on doing this, won't make non-portable project specific hacks that cannot be disabled, and distros can force it off in a consistent manner. Please only set these CGO_LDFLAGS when meson has been told to strip binaries.

Reply:
Based on this I've tidied up all of the go_env stuff.